pyodide_build/pywasmcross.py

```diff
@@ -118,6 +121,13 @@ def replay_genargs_handle_dashl(arg: str, used_libs: set[str]) -> str | None:
     if arg == "-lgfortran":
         return None
 
+    # Some Emscripten libraries that use setjmp/longjmp.
+    # The Emscripten linker should automatically know to use these variants so
+    # this shouldn't be necessary.
+    # This suffix will need to change soon to `-legacysjlj`.
+    if abi > "2025" and arg in ["-lfreetype", "-lpng"]:
+        arg += "-wasm-sjlj"
```

Comment:
How about checking the `abi` version more strictly, so that this code will not break when we update Emscripten version and update ABI version?

Reply:
I don't know of a more strict check that makes sense to me here. Are you saying because Emscripten changed the suffix to `"legacysjlj`? But ABI doesn't contain the Emscripten version. I think when we do that Emscripten update we update this too.

Reply:
For example, if we update the Emscripten version that changes the suffix + change the ABI version to 2025_1, this code will not work on that version. So I was curious if we can use `abi == 2025_0` here.

Reply:
Of course, otherwise we can set `max_pyodide_build_version` in pyodide-cross-build-environments.json instead, explicitly specifying that this version of pyodide-build will not be compatible for future Pyodide versions.

Reply:
I'm hoping to bring in that Emscripten update before we actually release. If you're concerned, perhaps I could add a check on the Emscripten version?

Reply:
> I'm hoping to bring in that Emscripten update before we actually release.

Oh, I see. This would work too.